<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QWidget" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>435</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="SearchLabel">
       <property name="text">
        <string>Filter by name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SearchBox">
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="FilterTypeLabel">
       <property name="text">
        <string>Filter by type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="FilterType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="SearchList"/>
   </item>
   <item>
     <widget class="QPushButton" name="InternetSearchButton">
       <property name="text">
         <string>or search Internet for (nothing)</string>
       </property>
     </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QComboBox</class>
   <extends>QComboBox</extends>
   <header>QComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>SearchBox</tabstop>
  <tabstop>FilterType</tabstop>
  <tabstop>SearchList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
